src/components/Progress/ProgressInnerText.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import {block} from '../utils/cn';
+
+const b = block('progress');
```

Comment:
Should move block init to constants like https://github.com/gravity-ui/uikit/blob/e9c092db83b3bdc76e5ab1586632a51b7e535348/src/components/Sheet/constants.ts